src/models/tools/geometry/geometry-content.ts

```diff
@@ -644,6 +676,30 @@ export const GeometryContentModel = types
       return properties;
     }
 
+    function getCommentAnchor(board: JXG.Board) {
+      const selectedObjects = self.selectedObjects(board);
+      if (selectedObjects.length === 1 && isPoint(selectedObjects[0])) {
+        return selectedObjects[0];
+      }
+
+      const selectedPolygon = selectedObjects.find(isPolygon);
```

Comment:
For points we only annotate if there is a single selected point. For polygons and lines we annotate the first selected object of the appropriate type. Seems like they should be consistent.